### ISSUE TITLE
[Config Bug]: New mask saving unquotes go2rtc ffmpeg lines

### ISSUE THREAD

**Issue by natevw** [blakeblackshear/frigate#8605]
### Describe the problem you are having

When saving a mask directly from 0.13.0-beta5 release, a section of configuration like

```
go2rtc:
    inornatus:
    - rtsp://speculor-inornatus.lan:554/1
    - "ffmpeg:inornatus#video=h264#hardware"
```

becomes instead

```
go2rtc:
    inornatus:
    - rtsp://speculor-inornatus.lan:554/1
    - ffmpeg:inornatus#video=h264#hardware
```

which I believe YAML treats as comments and discards the `#video=h264#hardware` options? The configuration update logic should probably check values for this (and any other strange [YAML gotchas](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#gotchas)) need for escaping?

### Version

0.13.0-AA93D4F

### Frigate config file

```yaml
(relevant sample provided above)
```

### Relevant log output

```shell
(don't think logs are relevant to this one?)
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

**Comment by natevw** [comment]
Technically maybe YAML requires a ` #` i.e. space first? As further testing indicates it does seem to get passed into go2rtc even without the quotes — maybe it's just the Web UI's `/config` code editor alone that makes it **look** like a comment even though the underlying YAML parser doesn't treat it as such?

**Comment by NickM-27** [comment]
Yes, it will just be the way the config editor displays it. We don't have control over how the yaml is formatted so this is working as expected
